Cargo.toml

```
@@ -2,7 +2,7 @@
 name = "ed25519-zebra"
 # Before publishing:
 # - update CHANGELOG.md
-version = "4.0.0"
+version = "4.0.1"
 rust-version = "1.65.0"
 authors = ["Henry de Valence <hdevalence@hdevalence.ca>"]
```

I think we missed adding ZF as an author here a long time ago.

```suggestion
authors = ["Henry de Valence <hdevalence@hdevalence.ca>", "Zcash Foundation <zebra@zfnd.org>"]
```

---

Our copyright is in the license files, so there's no legal impact here. But it helps to let people know.